src/main/java/seedu/wife/logic/commands/foodcommands/ViewCommand.java

```diff
@@ -0,0 +1,56 @@
+package seedu.wife.logic.commands.foodcommands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.wife.commons.core.Messages;
+import seedu.wife.commons.core.index.Index;
+import seedu.wife.logic.commands.Command;
+import seedu.wife.logic.commands.CommandResult;
+import seedu.wife.logic.commands.exceptions.CommandException;
+import seedu.wife.model.Model;
+import seedu.wife.model.food.Food;
+
+/**
+ * Views the food identified by the index number used in the displayed food list.
+ */
+public class ViewCommand extends Command {
+
+    public static final String COMMAND_WORD = "view";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Views the food identified by the index number used in the displayed food list.\n"
+            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_VIEW_FOOD_SUCCESS = "Viewed Food: %1$s";
+
+    private final Index targetIndex;
+
+    public ViewCommand(Index targetIndex) {
+        this.targetIndex = targetIndex;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+        List<Food> lastShownList = model.getFilteredFoodList();
+
+        if (targetIndex.getZeroBased() >= lastShownList.size()) {
+            throw new CommandException(Messages.MESSAGE_INVALID_FOOD_DISPLAYED_INDEX);
```

Comment:
Do you think it's better to have `MESSAGE_INVALID_FOOD_DISPLAYED_INDEX` imported with static import so it is consistent with line 46? 

Same issue exists for `ViewCommandTest`

Reply:
wdym by static import for that? it is the same functionality as delete though

Reply:
I meant like `import static seedu.wife.commons.core.Messages.MESSAGE_INVALID_FOOD_DISPLAYED_INDEX`, its more of keeping the code standardized but it is okay for now

Reply:
okay will keep that in mind next time